routes/web/project/location/postgres.rb

```diff
@@ -47,6 +47,25 @@ class CloverWeb
 
         r.redirect "#{@project.path}#{st.subject.path}"
       end
+
+      r.post "reset_superuser_password" do
```

Comment:
We use `-` as separator in general.
```suggestion
      r.post "reset-superuser-password" do
```